### PR TITLE
Rebuild How-To RNG WAX tutorial

### PR DESCRIPTION
**Pull request by 3dkrender**
The tutorial for the use of the WAX RNG smart contract has been rebuilt. 
Includes Spanish translation.

It is accompanied with a link to the source code of the complete project to follow the tutorial
https://github.com/3dkrender/WAX-RNG-Test

### CODE REVIEW

**Review by rakeden — APPROVED** [review]

**Comment by rakeden** [comment]
Thanks for the updated RNG tutorials and spanish translation
This PR belongs to #83

File: docs/en/tutorials/create-wax-rng-smart-contract/rng_sample.md [hunk 1]
```diff
@@ -1,141 +1,230 @@
...
+   // update table with random_value
+   _customers.modify(iCustomers, get_self(), [&](auto& rec) {
+      rec.random_value = random_value;
+      rec.final_number = num1;
+   });
+}
+```
+We make sure that only WAX RNG will be able to call this action with *require_auth*.
+
+We extract the first 8 bits of the returned random number and use it to get a number no larger than 100.
```
Comment (NKCSS) [inline docs/en/tutorials/create-wax-rng-smart-contract/rng_sample.md:212]: This isn't correct; they are just rebuilding a 64-bit uint 8 bits at a type (shift-left the sum by 8, then add the next byte in the index.

This is the same as 

`(byte_array[0] << 56) + (byte_array[1] << 48) + (byte_array[2] << 40) + (byte_array[3] << 32) + (byte_array[4] << 24) + (byte_array[5] << 16) + (byte_array[6] << 8) + byte_array[7];`

(just Imagine all the bytes are cast to uint64_t before shifting; left that out to reduce noise)

Comment (rakeden) [inline docs/en/tutorials/create-wax-rng-smart-contract/rng_sample.md:212]: @NKCSS This highlighted row?

**Review by NKCSS — COMMENTED** [review]
Found an issue in the text; since it's both English and Spanish, I try to report it this way to it can be updated (I don't speak spanish 😅).

File: docs/en/tutorials/create-wax-rng-smart-contract/rng_sample.md (hunk 1 above)
Comment (3dkrender) [reply docs/en/tutorials/create-wax-rng-smart-contract/rng_sample.md:212]: Yep. Sometimes we make the easy difficult and then we get lost.
This is the correction:
```c++
auto byte_array = random_value.extract_as_byte_array();
uint8_t random_int = 0;
random_int = byte_array[0];
uint64_t num1 = random_int % max_value;
```

Comment (rakeden) [reply docs/en/tutorials/create-wax-rng-smart-contract/rng_sample.md:212]: 🙏

**Pull request merged**